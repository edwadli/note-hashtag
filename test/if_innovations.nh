be PrintEndline "This revolutionary new language construct" unless false inwhichcase PrintEndline "Fail 1"
be (
  PrintEndline "is great for keeping programmers on their toes"
) unless false inwhichcase (
  PrintEndline "Fail 2"
)
