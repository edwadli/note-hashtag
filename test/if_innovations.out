This revolutionary new language construct
is great for keeping programmers on their toes
